app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -424,8 +423,10 @@ private void onDeleteClicked(Spinner spinner) {
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(s -> {
-                    DeleteTask deleteTask = new DeleteTask(getActivity(), media, reason);
-                    deleteTask.execute();
+                    if(null!=getActivity()) {
```

Comment:
```suggestion
                    if (getActivity() != null) {
```

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace